test/Libraries/DynamoPythonTests/DynamoPythonTests.csproj

```diff
@@ -1,4 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <UILib>true</UILib>
```

Comment:
? Is this coming from your other PR?

Reply:
Yes. I also removed this part https://github.com/DynamoDS/Dynamo/pull/14614/files#diff-bbde35c3169d8580248bcea0e6c8e5722086cba412a65c737fb4c8656d095c55L64 (Basically all test projects were UILibs automatically).